tools/isa-info.c

```diff
@@ -178,15 +178,20 @@ int main(int argc, char** argv) {
 		printf("\tPMULL: %s\n", cpuinfo_has_arm_pmull() ? "yes" : "no");
 		printf("\tCRC32: %s\n", cpuinfo_has_arm_crc32() ? "yes" : "no");
 #endif
+
 #if CPUINFO_ARCH_RISCV32 || CPUINFO_ARCH_RISCV64
 	printf("Instruction sets:\n");
-		printf("\tBase Integer: %s\n", cpuinfo_has_riscv_i() ? "yes" : "no");
-		printf("\tInteger Multiply/Divide: %s\n", cpuinfo_has_riscv_m() ? "yes" : "no");
-		printf("\tAtomics: %s\n", cpuinfo_has_riscv_a() ? "yes" : "no");
+		#if CPUINFO_ARCH_RISCV32
```

Comment:
Nested preprocessor statements should not be indented
```suggestion
#if CPUINFO_ARCH_RISCV32
```

Reply:
Did it